warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -328,6 +329,7 @@ public String getPlan(AccumuloClient client, Query settings, Set<Authorizations>
                             + (this.getSettings() == null ? "empty" : this.getSettings().getId()) + ')');
         this.config.setExpandFields(expandFields);
         this.config.setExpandValues(expandValues);
+        this.config.setExpandUnfieldedValues(expandValues);
```

Comment:
Add getter/setter for `expandUnfieldedValues`, I'm not sure if `deferPushdownPullup` requires a getter/setter here.

Reply:
Not sure we need it for expandUnfieldedValues, but I could see a case being made so I will add them.  Definitely not needed for deferPushdownPullup.